da/mock/mock.go

```diff
@@ -147,13 +147,14 @@ func (m *DataAvailabilityLayerClient) SubmitBlocks(ctx context.Context, blocks [
 
 	for _, block := range blocks {
 		blockHeight := uint64(block.Height())
-		m.logger.Debug("Submitting blocks to DA layer!", "height", blockHeight, "dataLayerHeight", daHeight)
```

Comment:
Assuming we can revert changes to this file once debugging is finished.

Reply:
reverted

Reply:
was the deletion the revert?

Reply:
yeah, whoops. brought it back.